addons/skin.estuary/xml/DialogPVRChannelGuide.xml

```diff
@@ -4,7 +4,9 @@
 	<controls>
 		<control type="group">
 			<depth>DepthOSD</depth>
+			<!--
 			<animation effect="fade" start="100" end="0" time="200" tween="sine" condition="$EXP[infodialog_active]">Conditional</animation>
+			-->
```

Comment:
it's part of the skip commit. I don't want to merge it as it is, because it needs a bit more love from @ronie or @phil65